src/procedures/proc_functions.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright FalkorDB Ltd. 2023 - present
+ * Licensed under the Server Side Public License v1 (SSPLv1).
+ */
+
+
+
+#include "../util/arr.h"
```

Comment:
For my OCD reasons, can you please order this list by line length, short -> long